src/cc/bcc_helpers.h

```diff
@@ -0,0 +1,53 @@
+#ifndef LIBBCC_ELF_H
```

Comment:
ifdef doesn't match file name. Should it be bcc_elf.h ?
bcc_helpers.h sounds too abstract and eventually other non-elf things
will start piling up in here.
Is it possible to add some simple c+c++ demo?
Everything so far is c+py or c+lua with focus of 'extern C' api
while c++ api is little bit of afterthought. 
With this new addition of elf api, something like offwaketime in c+c++ is now possible, would be really useful and it will help to stabilize c/c++ API.